apps/play/src/components/Accordion.tsx

```
@@ -0,0 +1,63 @@
+
+  return (
+    <div class='accordion'>
+      <For each={props.sections}>
+        {(section) => (
+          <div class='accordion-section'>
+            {section.title && (
+              <button
+                class={`accordion-header${openId() === section.id ? ' open' : ''}`}
+                onClick={() => handleToggle(section.id)}
+                aria-expanded={openId() === section.id}
+              >
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)